### DISCUSSION TITLE
Are we back?

### DISCUSSION THREAD

**Discussion by zethon** [zethon/Owl#111] [Announcements]
A couple of years ago, Conan switched repos and broke Owl's build system. At the time I didn't have the heart to fix it. Yet over the years I kept thinking about the project and played around with rewriting it from scratch, maybe even using something like Electron.

Recently, though, I started playing around with this source repo and got it to work! Though, I'm still not able to use Qt from a Conan repo, which annoys me.

The point is: Is this project alive again? The answer is a **definite maybe**!

**Comment by SamuraiCrow** [comment]
Are the Lua parsers available independently of Qt?

**Reply by zethon** [reply]
https://github.com/lulzapps/luaparsers

**Reply by SamuraiCrow** [reply]
Thanks!

**Reply by zethon** [reply]
I'm curious -- what did you want them for? I'm not even sure if they work anymore. :)

**Reply by SamuraiCrow** [reply]
I was thinking of using Owl-like programming but using a different framework on a niche operating system.  (Not Windows, not Mac, not Linux, not Android and not iOS)

**Reply by zethon** [reply]
I just saw your post in the other thread. It sounds like an interesting project. 

Unfortunately the Lua Parsers use a few custom-bindings in Lua, like the the `SGML` library for parsing HTML and Curl. I'm not sure if those are available on your system, but good luck!
